Add blog post for 4.3 RC 1 (#884)
godotengine/godot-website: collections/_article/release-candidate-godot-4-3-rc-1.md

---
title: "Release candidate: Godot 4.3 RC 1"

> excerpt: ""

Reviewer: ```suggestion
excerpt: "We are cautiously optimistic that Godot 4.3 is ready for release, please test it and let us know if we are right!"
```

categories: ["pre-release"]
author: "Rémi Verschelde"
image: /storage/blog/covers/release-candidate-godot-4-3-rc-1.webp
image_caption_title: "ColdRidge"
image_caption_description: "A game by Frog Collective"

> date: 2024-07-25 15:00:00

Reviewer 2: We need to start adding the proper time here, but it can be edited after the review. @winston-yallow do you know the local time of the server?

Author: Is it important for the time to be accurate? We've always picked roughly a time that's before publishing date on the same day, which seems good enough. I guess mostly RSS feeds might see it as "published 3 hours ago" even though it was published now.

As for local time, I'm assuming UTC here.

Still, if we can infer it at build time, that could work. But since we rebuild everything on each commit, we'd need that information to be committed somehow I guess?

More seriously, Godot is a big piece of software and it's hard for contributors and even unit tests to validate all areas of the engine when developing new features or bug fixes. So we rely on extensive testing from the community to find engine issues while testing dev, beta, and RC snapshots in your projects, and reporting them so that we can fix them prior to tagging the 4.3-stable release.

Please, consider [supporting the project financially](https://fund.godotengine.org), if you are able. Godot is maintained by the efforts of volunteers and a small team of paid contributors. Your donations go towards sponsoring their work and ensuring they can dedicate their undivided attention to the needs of the project.

[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.3.rc1/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.

---

> *The cover illustration is from* [**ColdRidge**](https://store.steampowered.com/app/3052500/ColdRidge/-), *a turn-based exploration game set in the mystical Wild West, developed by [Frog Collective](http://frog-collective.com/en/) with Godot 4. The game was revealed in early July and just got a playable demo [on Steam](https://store.steampowered.com/app/3052500/ColdRidge/), where you can also wishlist it.*

Reviewer 2: ```suggestion
*The cover illustration is from* [**ColdRidge**](https://store.steampowered.com/app/3052500/ColdRidge/), *a turn-based exploration game set in the mystical Wild West, developed by [Frog Collective](http://frog-collective.com/) with Godot 4. The game was revealed in early July and just got a playable demo [on Steam](https://store.steampowered.com/app/3052500/ColdRidge/), where you can also wishlist it.*
```

## Highlights

We covered the most important highlights from Godot 4.3 in the previous [4.3 beta 1 blog post](/article/dev-snapshot-godot-4-3-beta-1/), so if you haven't read that one, have a look to be introduced to the main new features added in the 4.3 release.

Especially if you're testing 4.3 for the first time, you'll want to get a condensed overview of what new features you might want to make use of.

> This section covers changes made since the previous [beta 3 snapshot](/article/dev-snapshot-godot-4-3-beta-3/), which are mostly regression fixes, or "safe" fixes to long standard issues.

Reviewer: ```suggestion
This section covers changes made since the previous [beta 3 snapshot](/article/dev-snapshot-godot-4-3-beta-3/), which are mostly regression fixes, or "safe" fixes to longstanding issues.
```

There are two big highlights in this release for the Windows platform though:

- The Direct3D 12 rendering backend now works out of the box on Windows, and no longer requires copying the `DXIL.dll` library ([GH-94203](https://github.com/godotengine/godot/pull/94203))! This is thanks to Microsoft's decision to [open source the DXIL Validator hash](https://devblogs.microsoft.com/directx/open-sourcing-dxil-validator-hash/), which was the only reason for us to require the proprietary `DXIL.dll`. Aside from greatly simplifying the distribution of Windows builds with D3D12, it also makes it possible to export Windows D3D12 projects from other OSes (Linux, macOS, Android), which is an important part of Godot's cross-platform support.

> - With the progressive rollout of ARM64 Windows devices on the market, we've finalized our support for that target and now provide ARM64 Windows builds (editor and export templates) for the first, for both Standard and .NET flavors. Thanks to Qualcomm engineers who did some testing and gave us access to cloud-based devices to confirm that the binaries work.

Reviewer: ```suggestion
- With the progressive rollout of ARM64 Windows devices on the market, we've finalized our support for that target and now provide ARM64 Windows builds (editor and export templates) for both Standard and .NET flavors. Thanks to Qualcomm engineers who did some testing and gave us access to cloud-based devices to confirm that the binaries work.
```
